docs/aks.md

```diff
@@ -0,0 +1,177 @@
+        customresourcedefinition.apiextensions.k8s.io/perconaservermongodbrestores.psmdb.percona.com serverside-applied
+        role.rbac.authorization.k8s.io/percona-server-mongodb-operator serverside-applied
+        serviceaccount/percona-server-mongodb-operator serverside-applied
+        rolebinding.rbac.authorization.k8s.io/service-account-percona-server-mongodb-operator serverside-applied
+        deployment.apps/percona-server-mongodb-operator serverside-applied
+        ```
+
+2. The operator has been started, and you can deploy your MongoDB cluster:
+
+    ```bash
+    $ kubectl apply -f https://raw.githubusercontent.com/percona/percona-server-mongodb-operator/v{{ release }}/deploy/cr.yaml
+    ```
+
+    ??? example "Expected output"
+
+        ```text
+        perconaservermongodb.psmdb.percona.com/my-cluster-name created
+        ```
+
+    !!! note
+
+        This deploys default MongoDB cluster configuration, one mongos node and
+        one config server node. Please see [deploy/cr.yaml](https://raw.githubusercontent.com/percona/percona-server-mongodb-operator/v{{ release }}/deploy/cr.yaml)
+        and [Custom Resource Options](operator.md#operator-custom-resource-options)
+        for the configuration options. You can clone the repository with all
+        manifests and source code by executing the following command:
+
+        ```bash
+        $ git clone -b v{{ release }} https://github.com/percona/percona-server-mongodb-operator
+        ```
+
+        After editing the needed options, apply your modified `deploy/cr.yaml` file as follows:
+
+        ```bash
+        $ kubectl apply -f deploy/cr.yaml
+        ```
+
+    The creation process may take some time. When the process is over your
+    cluster will obtain the `ready` status. You can check it with the following
+    command:
+
+    ```bash
+    $ kubectl get psmdb
+    ```
+
+    ??? example "Expected output"
+
+        ```text
+        NAME              ENDPOINT                                           STATUS   AGE
+        my-cluster-name   my-cluster-name-mongos.default.svc.cluster.local   ready    5m26s
+        ```
+
+## Verifying the cluster operation
+
+It may take ten minutes to get the cluster started. When `kubectl get psmdb`
+command finally shows you the cluster status as `ready`, you can try to connect
+to the cluster.
+
+{% include 'assets/fragments/connectivity.txt' %}
+
+## Troubleshooting
+
+If `kubectl get psmdb` command doesn't show `ready` status too long, you can 
+check the creation process with the `kubectl get pods` command:
+
+```bash
+$ kubectl get pods
+```
+
+??? example "Expected output"
+
+    --8<-- "./docs/assets/code/kubectl-get-pods-response.txt"
+
+If the command output had shown some errors, you can examine the problematic
+Pod with the `kubectl describe <pod name>` command as follows:
+
+```bash
+$ kubectl describe pod my-cluster-name-rs0-2
+```
```

Comment:
Let's add also 
`kubectl get psmdb <psmdb-name> -o yaml`
It also could be useful for troubleshooting.

Reply:
I'd propose placing this in a separate PR because this would be useful for all installation instructions